Introduce env vars to define idle state timeout (#271)
canonical/charmed-openstack-upgrader: docs/reference/environment-variables.rst

* **COU_TIMEOUT** - define timeout for **COU** retry policy. Default value is 10 seconds.
* **COU_MODEL_RETRIES** - define how many times to retry the connection to Juju model before giving up. Default value is 5 times.
* **COU_MODEL_RETRY_BACKOFF** - define number of seconds to increase the wait between connection to the Juju model retry attempts. Default value is 2 seconds.

> * **COU_STANDARD_IDLE_TIMEOUT** - define the idle timeout to check if the application has upgraded. Default value is 300 seconds.

Reviewer: ```suggestion
* **COU_STANDARD_IDLE_TIMEOUT** - how long COU will wait for an application to settle to active/idle and declare the upgrade complete. The default value is 300 seconds.
```

> * **COU_LONG_IDLE_TIMEOUT** - define the idle timeout for applications that take more time than usual to upgrade like, for example, Keystone and Octavia. Default value is 1800 seconds.

Reviewer 2: nit:
```suggestion
* **COU_LONG_IDLE_TIMEOUT** - define the idle timeout for applications that take more time than usual to upgrade such as Keystone and Octavia. Default value is 1800 seconds.
```

> * **COU_LONG_IDLE_TIMEOUT** - define the idle timeout for applications that take more time than usual to upgrade like, for example, Keystone and Octavia. Default value is 1800 seconds.

Reviewer: ```suggestion
* **COU_LONG_IDLE_TIMEOUT** - a longer version of COU_STANDARD_IDLE_TIMEOUT for applications that are known to need more time than usual to upgrade like, for example, Keystone and Octavia. The default value is 1800 seconds.
```
